src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -3,8 +3,9 @@
 import static java.lang.Boolean.TRUE;
 import static java.util.Objects.nonNull;
 import static java.util.Objects.requireNonNull;
-import static org.folio.circulation.domain.LoanAction.CHECKED_OUT;
+
```

Comment:
Please reorganise these imports to remove the gap between static imports

Reply:
DOne.